### PR TITLE
Interest Rate TWAP

### PR DESCRIPTION
**Pull request by Drypto13**

### CODE REVIEW

File: contracts/mixins/InterestHandler.sol [hunk 1]
```diff
@@ -147,8 +156,18 @@ contract InterestHandler is State, InterestRateEvents {
         returns (uint256 ratePerTokenNewAmount, uint256 nextInterestRate)
     {
         uint256 timeSinceUpdate;
+        uint32[] memory secondsAgo = new uint32[](2);
+        secondsAgo[0] = 0;
+        secondsAgo[1] = 3600;
```
Comment (RomanHiden) [inline contracts/mixins/InterestHandler.sol]: probably should use constant or `1 hour`

File: contracts/utils/InterestOracle.sol [hunk 2]
```diff
@@ -0,0 +1,291 @@
+pragma solidity ^0.5.0;
```
Comment (RomanHiden) [inline contracts/utils/InterestOracle.sol:1]: version shouldn't be that strict

**Review by RomanHiden — APPROVED** [review]
looks good

**Review by ghost — APPROVED** [review]
looks good

**Pull request merged**